AddOns/BasicFinancials/test/src/TestBasicBF.Codeunit.al

```diff
@@ -0,0 +1,301 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+codeunit 139502 "Test Basic BF"
```

Comment:
Test Basic Financials

Reply:
I have been prohibited to use that word, If you need more clarification then speak with Fredrik

Reply:
Ok, clarified. Keep it as is.